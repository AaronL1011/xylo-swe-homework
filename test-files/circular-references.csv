A1, B1 +, C1
A2, B2, A1 + C2
C1, A2 +, 1 1 +