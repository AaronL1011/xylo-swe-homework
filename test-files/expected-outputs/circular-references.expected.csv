#ERR,#ERR,#ERR
#ERR,#ERR,#ERR
#ERR,#ERR,2